tests/cpychecker/refcounts/uninitialized_data/function_arg/input.c: In function 'test':
tests/cpychecker/refcounts/uninitialized_data/function_arg/input.c:41:16: warning: passing uninitialized data (item) as argument 3 to function at tests/cpychecker/refcounts/uninitialized_data/function_arg/input.c:41 [enabled by default]
tests/cpychecker/refcounts/uninitialized_data/function_arg/input.c:34:12: note: when PyList_New() succeeds at:     result = PyList_New(len);
tests/cpychecker/refcounts/uninitialized_data/function_arg/input.c:35:8: note: taking False path at:     if (!result) {
tests/cpychecker/refcounts/uninitialized_data/function_arg/input.c:39:12: note: reaching:     for (i = 0; i < len; i++) {
tests/cpychecker/refcounts/uninitialized_data/function_arg/input.c:39:5: note: when considering range: 1 <= value <= 0x7fffffff at:     for (i = 0; i < len; i++) {
tests/cpychecker/refcounts/uninitialized_data/function_arg/input.c:39:5: note: taking True path at:     for (i = 0; i < len; i++) {
tests/cpychecker/refcounts/uninitialized_data/function_arg/input.c:41:16: note: reaching: 	PyList_SetItem(result, i, item);
tests/cpychecker/refcounts/uninitialized_data/function_arg/input.c:29:1: note: graphical error report for function 'test' written out to 'tests/cpychecker/refcounts/uninitialized_data/function_arg/input.c.test-refcount-errors.html'
